src/median.sol

```diff
@@ -61,10 +61,6 @@ contract Median is IMedian, Ownable {
         emit MinimumQuorumUpdated(_minimumQuorum);
     }
 
-    function read() external view returns (uint256, uint256) {
```

Comment:
what replaces this? @AmadiMichael

Reply:
This [line](https://github.com/Descent-Collective/oracle-module/blob/64205800b7ae405c4517820c9e7e4510580c0b30/src/median.sol#L13)

note that the median is never read directly. neither is the osm. the median is what relayers update, then the osm enforces a delay per top of the hour and then the feed now reads the osm and uses the value to update the vault.

Reply:
got it